wazo_confd/plugins/ha/service.py

```diff
@@ -1,14 +1,43 @@
-# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2019-2020 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from sqlalchemy.orm.session import make_transient
+
 
 class HAService:
-    def __init__(self, notifier, sysconfd):
+    def __init__(self, sip_general_service, registrar_service, notifier, sysconfd):
         self.notifier = notifier
+        self.registrar_service = registrar_service
+        self.sip_general_service = sip_general_service
         self.sysconfd = sysconfd
 
     def get(self):
         return self.sysconfd.get_ha_config()
 
     def edit(self, form):
+        if form['node_type'] == 'disabled':
+            self._update_sip_general_options(
+                {'minexpiry': '60', 'maxexpiry': '3600', 'defaultexpiry': '120'}
```

Comment:
Is it common for an admin to set different values for these options?

Reply:
Maybe, but they're necessary for high-availability to function correctly, so we just overwrite them.